Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM node:lts
+WORKDIR /app
+COPY package.json /app
+RUN npm install
+COPY . /app
+CMD ["npm", "start"]
```

Comment:
This would probably be better as `ENTRYPOINT ["npm", "run", "start", "--"]` to make additional arguments work correctly.

Reply:
I feel it would be better as `ENTRYPOINT ["npm", "run",  "--"]` rather than having the `start` argument there

Reply:
Used `ENTRYPOINT` and made a few changes to accommodate `nodemon` live changes too